turbopack/crates/turbopack-dev-server/src/lib.rs

```diff
@@ -173,6 +173,8 @@ impl DevServerBuilder {
                             uri: request.uri().clone(),
                         };
                         run_once_with_reason(tt.clone(), reason, async move {
+                            // TODO: `get_issue_reporter` should be an `OperationVc`, as there's a
+                            // risk it could be a task-local Vc, which is not safe for us to await.
                             let issue_reporter = get_issue_reporter();
```

Comment:
IMO this is pretty bad, but the `issue_reporter` `Vc` is never read (there's an `#[allow(dead_code)]`, and this isn't part of the nextjs codepath, so I just left a comment in case this blows up later.